libnavigation-core/src/test/java/com/mapbox/navigation/core/routerefresh/RouteRefreshControllerTest.kt

```diff
@@ -597,6 +615,11 @@ class RouteRefreshControllerTest {
                 currentRoute.directionsRoute.legs()!![0].incidents(),
                 refreshedRoute.directionsRoute.legs()!![0].incidents()!!
             )
+            assertEquals(
+                "closures on passed legs should not be refreshed",
+                currentRoute.directionsRoute.legs()!![0].closures(),
+                refreshedRoute.directionsRoute.legs()!![0].closures()!!
+            )
```

Comment:
```suggestion
            assertEquals(
                "closures on the route should not be refreshed",
                currentRoute.directionsRoute.legs()!!.map { it.closures() },
                refreshedRoute.directionsRoute.legs()!!.map { it.closures() }
            )
```

Reply:
Could you also add documentation for `RouteRefreshOptions` that the congestion and expired incidents are removed (and mention the interval), and that closures are not?

cc @VysotskiVadim

Reply:
@LukasPaczos @VysotskiVadim wdyt
```md

/**
 * The options available for refreshing the active [DirectionsRoute]. Each refresh will update
 * the current route's [LegAnnotation]. This includes traffic congestion and estimated travel time.
 *
 * In case if a route refresh is failing incidents and congestions (traffic tendency) annotations
 * can be removed from the route (see conditions below). Closures are removed if they were
 * clean upped via success route refresh only.
 *
 * Incidents are removed from a route:
 * - route refresh request is failing for 3*[intervalMillis] (default is 15 min) and the [Incident]
 * reached `Incident#endTime` (if non-null).
 *
 * Congestion (traffic tendency) annotations are removed from a route:
 * - route refresh request is failing for 3*[intervalMillis] (default is 15 min).
 *
 * Make sure that [RouteOptions.enableRefresh] is true to take advantage of this feature.
 *
 * @param intervalMillis The refresh interval in milliseconds, default is 5 min.
 */
class RouteRefreshOptions private constructor(
    val intervalMillis: Long
) {
...
```

Reply:
I realised that `RouteRefreshController` doesn't cleanup traffic tendency while I was reading the documentation 😱

Reply:
@VysotskiVadim what do you mean?

Reply:
oh, now I see